lib/browser/remote/objects-registry.ts

```diff
@@ -1,15 +1,18 @@
 'use strict'
 
+import { WebContents } from 'electron'
+
 const v8Util = process.electronBinding('v8_util')
 
-const getOwnerKey = (webContents, contextId) => {
+const getOwnerKey = (webContents: WebContents, contextId: string) => {
   return `${webContents.id}-${contextId}`
 }
 
 class ObjectsRegistry {
+  nextId: number = 0
```

Comment:
This can/should be private?

```suggestion
  private nextId: number = 0
```